CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # Changelog
 
+## v0.9.0 - 2022-04-03
+
+This is a major update to Shoehorn that includes breaking changes:
+
+* The `:init` configuration option only supports applications now. MFAs are no
+  longer supported and moved to `runtime.exs` or an `Application.start`
+  callback.
+* References to `use Shoehorn.Handler` need to be updated to `@behaviour
+  Shoehorn.Handler`. This may require implementing additional functions.
+* Elixir 1.9 is no longer supported. Please update to Elixir 1.10 or later.
+
+The main update to Shoehorn is to move all application startup to the boot
+script. This noticeably improves boot time on many Nerves platforms due to boot
+scripts being able to load files without traversing the entire Erlang module
+path list. These traversals are amazingly slow (sometimes seconds) due to a
+combination of SquashFS slowness in this area and slow overall IO.
+
+Using boot scripts to load all applications has some important improvements in
+addition to performance:
+
+* Application start order is deterministic and computed at compile-time. If you
+  want to see the order, take a look at the end of the `shoehorn.script` in your
+  release directory.
+* Shoehorn alphabetizes the start of applications that could be ordered
+  arbitrarily. This minimizes changes in start ordering when dependencies are
+  added or removed.
+* It enables experimental features like providing additional dependencies (using
+  the `:deps` configuration key) or hinting that dependencies get started as
```

Comment:
`:deps` looks totally new to me. Is it supposed to be `:extra_dependencies`?

Reply:
I probably should just delete it. It's a bonus feature that I put in so that I could experiment with it. The feature option is `:deps`, but now that you mention it, `:extra_dependencies` may be a better name.

Reply:
Oops.... you're right.

Reply:
Fixed. Thanks!